charts/incubator/blocky/values.yaml

```diff
@@ -1,10 +1,14 @@
 image:
-  repository: spx01/blocky
-  tag: development@sha256:ddb35986cbc924de11cd37ccf625ff6bd0896fad456e57ee9c0bd67bd034770e
+  repository: tccr.io/truecharts/alpine
+  tag: 3.16.2@sha256:71198b13ca5eaa344c6bd2bd0c1a29fb084772433c2811bcf9a78b77838b87fa
```

Comment:
huh?

Reply:
you can't do `ls` , `cat` etc with blocky.
Wanted to see some files inside

Reply:
ahh okey